contracts/BaseFreezeVoting.sol

```diff
@@ -18,75 +37,78 @@ abstract contract BaseFreezeVoting is FactoryFriendly, IBaseFreezeVoting {
     event FreezePeriodUpdated(uint256 freezePeriod);
     event FreezeProposalPeriodUpdated(uint256 freezeProposalPeriod);
 
-    /// @notice Allows user to cast a freeze vote, creating a freeze proposal if necessary
+    /**
+     * Casts a positive vote to freeze the subDAO. This function is intended to be called
+     * by the individual token holders themselves directly, and will allot their token
+     * holdings a "yes" votes towards freezing.
+     *
+     * Additionally, if a vote to freeze is not already running, calling this will initiate
+     * a new vote to freeze it.
+     */
     function castFreezeVote() external virtual;
 
-    /// @notice Unfreezes the DAO, only callable by the owner
+    /**
+     * Unfreezes the DAO, only callable by the owner (parentDAO).
+     */
     function unfreeze() external onlyOwner {
         freezeProposalCreatedBlock = 0;
         freezeProposalVoteCount = 0;
     }
 
-    /// @notice Updates the freeze votes threshold, only callable by the owner
-    /// @param _freezeVotesThreshold Number of freeze votes required to activate a freeze
-    function updateFreezeVotesThreshold(
-        uint256 _freezeVotesThreshold
-    ) external onlyOwner {
+    /**
+     * Updates the freeze votes threshold, the number of votes required to enact a freeze.
+     *
+     * @param _freezeVotesThreshold Number of freeze votes required to activate a freeze
+     */
+    function updateFreezeVotesThreshold(uint256 _freezeVotesThreshold) external onlyOwner {
         _updateFreezeVotesThreshold(_freezeVotesThreshold);
     }
 
-    /// @notice Updates the freeze proposal period, only callable by the owner
-    /// @param _freezeProposalPeriod The number of blocks a freeze proposal has to succeed
-    function updateFreezeProposalPeriod(
-        uint256 _freezeProposalPeriod
-    ) external onlyOwner {
-        _updateFreezeProposalPeriod(_freezeProposalPeriod);
-    }
-
-    /// @notice Updates the freeze period, only callable by the owner
-    /// @param _freezePeriod The number of blocks a freeze lasts, from time of freeze proposal creation
-    function updateFreezePeriod(uint256 _freezePeriod) external onlyOwner {
-        _updateFreezePeriod(_freezePeriod);
-    }
-
-    /// @notice Updates the freeze votes threshold
-    /// @param _freezeVotesThreshold Number of freeze votes required to activate a freeze
-    function _updateFreezeVotesThreshold(
-        uint256 _freezeVotesThreshold
-    ) internal {
```

Comment:
One of the recommended solidity conventions I've been trying to follow is the function ordering, which places external functions before internal: 

https://docs.soliditylang.org/en/develop/style-guide.html#order-of-functions

Reply:
@tbwebb22 updated, thanks!